conference/react-js/src/reducers/talks.js

```diff
@@ -9,19 +9,20 @@ const talks = (state = [], action) => {
     return payload;
   } else if (type === APPLY_FILTERS) {
     const {
-      filter: { conferenceName, title, status },
+      filter: { name, title, topic, status },
       talks: listOfTalks,
       quantity,
       page,
     } = payload;
     const prevValue = (page - 1) * quantity;
     const lastValue = page * quantity;
     const filtered = (elementOfListTopic) => {
-      const countConference = elementOfListTopic.conferenceName
-        .indexOf(conferenceName);
-      const countTitle = elementOfListTopic.title.indexOf(title);
-      const countStatus = elementOfListTopic.status.indexOf(status);
-      return countConference > -1 && countTitle > -1 && countStatus > -1;
+      const countName = elementOfListTopic.name.toLowerCase().indexOf(name.toLowerCase());
+      const countTitle = elementOfListTopic.title.toLowerCase().indexOf(title.toLowerCase());
+      const countTopic = elementOfListTopic.topic.toLowerCase().indexOf(topic.toLowerCase());
+      const countStatus = elementOfListTopic.status.toLowerCase().indexOf(status.toLowerCase());
+      return countName > -1 && countTitle > -1 && countStatus > -1 &&
+      countTopic > -1;
```

Comment:
to improve readability you may use
[includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) method.
this piece of code looks quite complex